windows/src/engine/keyman32/appcontext.cpp

```diff
@@ -164,3 +164,36 @@ ContextItemToAppContext(km_core_context_item *contextItems, PWSTR outBuf, DWORD
   delete[] buf;
   return TRUE;
 }
+
+FormatContextResult
+format_context_for_core(LPCWSTR windows_context, LPWSTR core_context, uint32_t output_size) {
```

Comment:
Can we rename `output_size` to `core_context_size_in_chars` or something like that?